test/broadway_rabbitmq/ampq_client_test.exs

```diff
@@ -45,6 +45,19 @@ defmodule BroadwayRabbitMQ.AmqpClientTest do
                 %{connection: connection, qos: qos, requeue: :once, metadata: metadata}}
     end
 
+    test "configure connection via uri" do
+      connection = "amqp://guest:guest@127.0.0.1"
+
+      assert AmqpClient.init(queue: "queue", connection: connection) ==
+               {:ok, "queue",
+                %{
+                  connection: connection,
+                  qos: [prefetch_count: 50],
+                  requeue: :always,
+                  metadata: []
+                }}
```

Comment:
@polymetis since you've already had to change this test because of other changes in options, it might be more maintainable to test like this:

```elixir
    assert {:ok, "queue", %{connection: connection}} = AmqpClient.init(queue: "queue", connection: connection)
```

Reply:
Much better :) https://github.com/koan-ci/broadway_rabbitmq/pull/1/commits/9edbce990b001400dd4ced40c795043dbdb4d8b5